core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -789,6 +810,23 @@ private[spark] class ExecutorAllocationManager(
       }
     }
 
+    override def onUnschedulableTaskSet
+      (unschedulableTaskSet: SparkListenerUnschedulableTaskSet): Unit = {
```

Comment:
nit, fix indentation and style. "(" should be on previous line, parameters indented 4 spaces